packages/button/__tests__/Button.test.tsx

```diff
@@ -0,0 +1,49 @@
+import { render, screen } from '@testing-library/react'
+
+import Button from '../src/'
+import type { ButtonProps } from '../src/'
+
+it('renders button with no props', () => {
+  render(<Button />)
+  // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+  // @ts-ignore
+  expect(screen.getByRole('button')).toBeInTheDocument()
```

Comment:
Not sure why I got this test error:

```
__tests__/Button.test.tsx:8:27 - error TS2345: Argument of type '"button"' is not assignable to parameter of type 'ByRoleOptions | undefined'.

8   expect(screen.getByRole('button')).toBeInTheDocument()
```

I checked the types and docs and the usage is correct here. I ignored it since I didn't want to go down a rabbit hole